tests/console/curl_ipv6.pm

```diff
@@ -21,9 +21,13 @@ use strict;
 use warnings;
 
 sub run {
-    select_console 'user-console';
+    select_console 'root-console';
```

Comment:
hm, I don't think we should do it right away. As the description explicit mention
```
- switch to normal user
```
Maybe we need to ask @okurz ?

Reply:
Or I can revert this change and just skip this test as my task is 'to have something' for now.

I am also planning to move the tunnel from `user-console` to background so then we can re-enable this.